docs/integrations/index.md

```diff
@@ -0,0 +1,35 @@
+---
+id: index
+title: Integrations
+sidebar_label: Overview
+description:
+  Configuring Backstage to read or publish data with external providers using
+  integrations
+---
+
+Integrations allow Backstage to read or publish data using external providers
+
+- such as GitHub, GitLab, BitBucket, LDAP, or cloud providers.
```

Comment:
This one looks a bit funny.  Maybe for now we could just go without the bullet and leave it inline? Since it's just a rough example, not a comprehensively list, that may be easier to read (and not need to maintain over time).

![image](https://user-images.githubusercontent.com/33203301/112352190-f2237c80-8ca0-11eb-9406-0c1a77721c91.png)

Reply:
Oh wow, accidental bullet 😂  Nice catch!